030-Data-types/030-data-model.mdx

```diff
@@ -7,27 +7,27 @@ slug: concepts/data-model
 
 ## Xata specific data types
 
 Below is a mapping of the additional data types Xata provides alongside Postgres types.
 
-| Xata data type          | PostgreSQL data type | Notes                                                   |
-| ----------------------- | -------------------- | ------------------------------------------------------- |
-| [string](#string)       | text                 | With constraints on maximum length of 2048 characters   |
-| [text](#text)           | text                 | With constraints on maximum length of 204800 characters |
-| [int](#int)             | bigint               |                                                         |
-| [float](#float)         | double precision     |                                                         |
-| [datetime](#datetime)   | timestamptz          |                                                         |
-| [bool](#bool)           | boolean              |                                                         |
-| [email](#email)         | text                 | Validated by Xata                                       |
-| [multiple](#multiple)   | text[]               | Validated by Xata                                       |
-| [file](#file)           | xata.xata_file       | json object stored natively as jsonb                    |
-| [filearray](#filearray) | xata.xata_file_array | json array stored natively as jsonb                     |
-| [link](#link)           | foreign key          |                                                         |
-| [vector](#vector)       | real[]               |                                                         |
-| [json](#json)           | jsonb                |                                                         |
+| Xata data type                      | PostgreSQL data type | Notes                                                 |
+| ----------------------------------- | -------------------- | ----------------------------------------------------- |
+| [text](#text)                       | text                 |                                                       |
+| [string](#string) (deprecated)      | text                 | With constraints on maximum length of 2048 characters |
+| [int](#int)                         | bigint               |                                                       |
+| [float](#float)                     | double precision     |                                                       |
+| [timestamptz](#timestamptz)         | timestamptz          |                                                       |
```

Comment:
I wonder if we should communicate it as both names (datetime/timestamptz) since the CLI for e.g. will still use datetime for things like edit

Reply:
agreed, added fix